### PR TITLE
promote update of spraproxy repo form staging to production

### PR DESCRIPTION
**Pull request by jkhelil**
fixes https://issues.redhat.com/browse/PLNSRVCE-1602
Promote https://github.com/redhat-appstudio/infra-deployments/pull/3224 to production

### CODE REVIEW

File: components/sprayproxy/production/kustomization.yaml [hunk 1]
```diff
@@ -2,13 +2,13 @@ apiVersion: kustomize.config.k8s.io/v1beta1
...
 resources:
   - ../base
-  - https://github.com/redhat-appstudio/sprayproxy/config?ref=c6f8bf26007c140d51326760e16eb52ae5072e3b
+  - https://github.com/konflux-ci/sprayproxy/config?ref=53ae76d32a32be7babf32df376b89d0623e78726
   - pipelines-as-code-secret.yaml
 
 images:
-  - name: ko://github.com/redhat-appstudio/sprayproxy
+  - name: ko://github.com/konflux-ci/sprayproxy
     newName: quay.io/redhat-appstudio/sprayproxy
```
Comment (hugares) [inline components/sprayproxy/production/kustomization.yaml:10]: Is it possible newName must also use new org name?  The pod cannot start on staging: Back-off pulling image
            "quay.io/redhat-appstudio/sprayproxy:53ae76d32a32be7babf32df376b89d0623e78726"

Comment (jkhelil) [reply components/sprayproxy/production/kustomization.yaml:10]: the image doesnt exist yet, with this sha, I am trying to ask folks how images re built for sprayproxy

Comment (jkhelil) [reply components/sprayproxy/production/kustomization.yaml:10]: the repo is missing the github app and hence the ci is not running and image not created
Asking on forum-konflux-devlopper if someone can add the github app

**Comment by enarha** [comment]
/lgtm
The same commit and tag are using in staging and the service in staging looks healthy.

**Review by Roming22 — APPROVED** [review]
/lgtm

**Pull request merged**